_includes/scripts.html

```diff
@@ -30,16 +30,27 @@ <h5>Popular tags: </h5>
     </ul>
   </div>
   <table class="projects">
-   <% _.each(projects, function(project){ %>
-    <tbody>
+   <% _.each(projects, function(project) { %>
+    <tbody class="<% if ( project.stats ) { %>counted<% } %>">
```

Comment:
We need to set this `class` value to opt-out of querying the GitHub API:

https://github.com/up-for-grabs/up-for-grabs.net/blob/4ded32b6a4af1fd11217f28b22c469810a4d3211/javascripts/main.js#L254-L260